exonum-java-binding-common/src/main/java/com/exonum/binding/common/proofs/common/CheckedProof.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2018 The Exonum Team
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.exonum.binding.common.proofs.common;
+
+import com.exonum.binding.common.hash.HashCode;
+
+/**
+ * Common interface for CheckedProof operations.
```

Comment:
Please specify what that interface represents:
A checked proof is a result of proof verification operation.
If it is valid, the proof contents may be accessed.